core/src/main/java/org/infinispan/container/impl/DefaultDataContainer.java

```diff
@@ -233,4 +232,10 @@ public void cleanUp() {
          evictionCache.cleanUp();
       }
    }
+
+
+   @Override
+   public void forEachSegment(ObjIntConsumer<PeekableTouchableMap<K, V>> segmentMapConsumer) {
+      segmentMapConsumer.accept(entries, 0);
```

Comment:
The point is to iterate over all the various segmented maps of the container. I can't see having the API not have the segment.

And the reason for 0 is because this container has a single map. This is inline with what the `SingleSegmentKeyPartitioner` does as well.